closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -1016,8 +1020,13 @@ private ProfileParser(String id, Properties props) {
 
             parsedProfiles = new HashSet<>();
 
+            profileCheckPropertyNames = new LinkedList<>();
+            profileCheckProviderNames = new LinkedList<>();
```

Comment:
Suggest `HashSet`; remove unused import for `LinkedList`.

Reply:
Done.